samples/apps/copilot-chat-app/webapp/src/components/chat/typing-indicator/TypingIndicatorRenderer.tsx

```diff
@@ -14,14 +12,35 @@ const useClasses = makeStyles({
     },
 });
 
-export const TypingIndicatorRenderer: React.FC = () => {
-    // TODO: Make this stateless React component. No need to connect to app state.
-    const { conversations, selectedId } = useAppSelector((state: RootState) => state.conversations);
+interface TypingIndicatorRendererProps {
+    isBotTyping: boolean;
+    numberOfUsersTyping: number;
+}
+
+export const TypingIndicatorRenderer: React.FC<TypingIndicatorRendererProps> = (props) => {
+    const { isBotTyping, numberOfUsersTyping } = props;
     const classes = useClasses();
 
+    let message = "";
+    if (isBotTyping && numberOfUsersTyping > 0) {
+        message = `Bot and ${numberOfUsersTyping} user(s) are typing`;
```

Comment:
Can you add the same if else from below here, and make the message more specific

        if (numberOfUsersTyping === 1) {
            message = `Bot and 1 user istyping`;
        } else {
            message = `Bot and ${numberOfUsersTyping} users are typing`;
        }